src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1914,16 +1904,6 @@ namespace cryptonote
     m_quorum_cop.set_votes_relayed(votes);
   }
   //-----------------------------------------------------------------------------------------------
-  bool core::get_block_template(block& b, const account_public_address& adr, difficulty_type& diffic, uint64_t& height, uint64_t& expected_reward, const blobdata& ex_nonce)
```

Comment:
Was there a reason this was moved to the bottom?

Reply:
I think I had moved it out only to move it back in, I will revert the move